DOCS-1889: Mentions that PayPal is non-collecting and consequences (#1001)
MultiSafepay/docs: content/tools/reports/accountant-report-export.md

In the report, all (processed) incoming and outgoing completed transactions are displayed in the specified period in which you generate the report, providing you insights on an individual level, as well as of all the associated costs.

> __MultiSafepay does not collect funds for PayPal and therefore transactions will not appear in the accountant export. You must ensure that the financial status is set to _Completed_ in your backend. This will then be reflected in the accountant export.__

Reviewer: ```suggestion
__MultiSafepay does not collect funds for PayPal. Therefore, PayPal transactions do not appear in the accountant export. Make sure that the financial status is set to _Completed_ in your backend. This will then be reflected in the accountant export.__
```
